grobid-home/models/acknowledgment/TrainResult.txt

```diff
@@ -1,401 +0,0 @@
-(base) MP16-PRO:grobid tkristan$ ./gradlew train_acknowledgment
```

Comment:
this seems the same as the evaluation result - should remove one of the two - ideally this one